src/main/java/nayana/Ui.java

```diff
@@ -0,0 +1,153 @@
+package nayana;
+import java.util.ArrayList;
+import java.util.Scanner;
+import nayana.task.*;
```

Comment:
Similar wildcard imports are found in multiple other classes, I think they should be replaced to comply with code standards?